HHS/simpler-grants-gov
# [Issue 2783] indicate active nav item

## Summary
Fixes #2783

### Time to review: __5 mins__

## Changes proposed
Adds indicator to active nav item

## Context for reviewers
### Test steps

1. visit the local site on the branch and click on nav items in mobile and desktop widths
2. _VERIFY_: an indicator appears below the item matching the currently active route

## Additional information

### Desktop
<img width="1132" alt="Screenshot 2024-11-15 at 1 27 44 PM" src="https://github.com/user-attachments/assets/bc0c508c-7581-4914-a882-ea1b9341c3ec">


### Mobile
<img width="775" alt="Screenshot 2024-11-15 at 1 27 58 PM" src="https://github.com/user-attachments/assets/b86d0eba-a60e-492e-8b1e-ba1e3d0d5430">
